eo-runtime/src/test/eo/org/eolang/negative-infinity-tests.eo

```diff
@@ -27,171 +27,367 @@
 +junit
 +version 0.0.0
 
-[] > negative-infinity-eq-is-valid
+# Equal to
+[] > negative-infinity-is-equal-to-one-div-zero
+  assert-that > @
+    negative-infinity
+    $.equal-to (-1.0.div 0.0)
+
+[] > negative-infinity-eq-negative-infinity
   negative-infinity > neg-inf!
+  assert-that > @
+    neg-inf
+    $.equal-to neg-inf
 
+[] > negative-infinity-not-eq-positive-infinity
   assert-that > @
-    and.
-      neg-inf.eq neg-inf
-      (neg-inf.eq positive-infinity).eq FALSE
-      (neg-inf.eq nan).eq FALSE
-      (neg-inf.eq 42).eq FALSE
-      (neg-inf.eq 42.5).eq FALSE
-    $.equal-to TRUE
+    negative-infinity
+    $.not
+      $.equal-to positive-infinity
+
+[] > negative-infinity-not-eq-nan
+  assert-that > @
+    negative-infinity
+    $.not
+      $.equal-to nan
+
+[] > negative-infinity-not-eq-int
```

Comment:
@maxonfjvipon Why did you decide to split one test to a small ones?

Reply:
@Graur for some reason they causes stackoverflow exception

Reply:
@maxonfjvipon Looks like an issue in `eo-hamcrest`. Thanks!